mungegithub/mungers/submit-queue.go

```diff
@@ -381,8 +383,8 @@ func (sq *SubmitQueue) internalInitialize(config *github.Config, features *featu
 			gcs = utils.NewTestUtils("bucket", "logs", overrideUrl)
 		} else {
 			gcs = utils.NewWithPresubmitDetection(
-				utils.KubekinsBucket, utils.LogDir,
-				utils.PullKey, utils.PullLogDir,
+				sq.features.GCSInfo.BucketName, sq.features.GCSInfo.LogDir,
+				sq.features.GCSInfo.PullKey, sq.features.GCSInfo.PullLogDir,
```

Comment:
I assume this is the  main part of the feature: we want to be able to change these value. Does it need to be a feature though?